### PR TITLE
Use XXXX_ca_b64 for an encoded cert

### PR DESCRIPTION
**Pull request by sayf-eddine-scality**
Instead of XXXX_ca_server

Fixes: #890

**Component**:

**Context**: 

**Summary**:

**Acceptance criteria**: 

---

Closes: #ISSUE_NUMBER

### CODE REVIEW

File: salt/metalk8s/kubernetes/ca/kubernetes/advertised.sls [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{%- set ca_server = salt['mine.get'](pillar['metalk8s']['ca']['minion'], 'kubernetes_ca_server') %}
+{%- set ca_server = salt['mine.get'](pillar['metalk8s']['ca']['minion'], 'kubernetes_root_ca_b64') %}
```
Comment (TeddyAndrieux) [inline salt/metalk8s/kubernetes/ca/kubernetes/advertised.sls]: I think it's make sense to also rename this `ca_server` variable here

File: pillar/metalk8s/roles/ca.sls [hunk 2]
```diff
@@ -1,5 +1,5 @@
 mine_functions:
-  kubernetes_ca_server:
+  kubernetes_root_ca_b64:
```
Comment (slaperche-scality) [inline pillar/metalk8s/roles/ca.sls:2]: The commit message doesn't explain WHY we do the renaming.

**Review by slaperche-scality — CHANGES_REQUESTED** [review]

**Review by slaperche-scality — APPROVED** [review]

**Review by TeddyAndrieux — APPROVED** [review]
LGTM

**Pull request merged**